app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.tsx

```diff
@@ -82,7 +82,9 @@ function DocumentCaptureTroubleshootingOptions({
       {hasErrors && inPersonURL && showInPersonOption && (
         <TroubleshootingOptions
           isNewFeatures
-          heading={t('idv.troubleshooting.headings.are_you_near')}
+          heading={formatHTML(t('idv.troubleshooting.headings.are_you_near'), {
+            wbr: 'wbr',
+          })}
```

Comment:
It appears that the `heading` prop uses `string` as the TypeScript type, which isn't compatible with the return value of `formatHTML` (`ReactNode`). I think it would be straightforward/valid to update the `string` type to `ReactNode` to indicate that the prop can be anything that React can render.

Updating the type on this line from `string` to `ReactNode` (already imported in the file):

https://github.com/18F/identity-idp/blob/7415a07209b00f013c4b942683bea27a1c3cc3cf/app/javascript/packages/components/troubleshooting-options.tsx#L17

Original error:

```
$ yarn typecheck
yarn run v1.22.19
$ tsc
app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.tsx:85:11 - error TS2322: Type 'ReactNode' is not assignable to type 'string | undefined'.
  Type 'null' is not assignable to type 'string | undefined'.

85           heading={formatHTML(t('idv.troubleshooting.headings.are_you_near'), {
             ~~~~~~~

  app/javascript/packages/components/troubleshooting-options.tsx:17:3
    17   heading?: string;
         ~~~~~~~
    The expected type comes from property 'heading' which is declared here on type 'IntrinsicAttributes & TroubleshootingOptionsProps'


Found 1 error in app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.tsx:85

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```